tools/first-address-from-root/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "derive-master-public-key"
```

Comment:
so shall we call it `first address from root` or `derive master public key`

Reply:
right.. I actually don't like neither, it's like the fourth name I try. Any suggestions? Otherwise, `first-address-from-root` I guess, but it's a bit annoying to type.

Reply:
https://github.com/dcSpark/dcspark-core/pull/19#discussion_r1042651003